Libraries/Inspector/InspectorPanel.js

```diff
@@ -14,14 +14,43 @@ const ElementProperties = require('ElementProperties');
 const NetworkOverlay = require('NetworkOverlay');
 const PerformanceOverlay = require('PerformanceOverlay');
 const React = require('React');
-const PropTypes = require('prop-types');
 const ScrollView = require('ScrollView');
 const StyleSheet = require('StyleSheet');
 const Text = require('Text');
 const TouchableHighlight = require('TouchableHighlight');
 const View = require('View');
 
-class InspectorPanel extends React.Component<$FlowFixMeProps> {
+import type {ViewStyleProp} from 'StyleSheet';
+
+type Props = $ReadOnly<{|
+  devtoolsIsOpen?: ?boolean,
+  inspecting?: ?boolean,
+  setInspecting?: ?(val: boolean) => void,
+  perfing?: ?boolean,
+  setPerfing?: ?(val: boolean) => void,
+  touchTargeting?: ?boolean,
+  setTouchTargeting?: ?(val: boolean) => void,
+  networking?: ?boolean,
+  setNetworking?: ?(val: boolean) => void,
+  hierarchy?: any,
+  selection?: ?number,
+  setSelection?: number => mixed,
+  inspected?: ?{
+    style?: ?ViewStyleProp,
+    frame?: ?{
+      top?: ?number,
+      left?: ?number,
+      width?: ?number,
+      height: ?number,
+    },
+    source?: ?{
+      fileName?: string,
+      lineNumber?: number,
+    },
+  },
```

Comment:
Cool. Also, I found these types from other components. So should we export these types on those components and reference them here? There are many components in Inspector which needs to be converted to flow types.

Reply:
Yeah, that sounds like a great idea! I can see there being an `InspectorTypes` module under `Libraries/Inspector` that contains shared types for the modules in that folder. But I think it may be too much to make that refactor in this diff. 

Would you like to tackle this in another PR?

Reply:
Sure. I'll finish up with this today and work on the Inspector folder next week.